js/techtree.js

```diff
@@ -337,6 +337,17 @@ const ELITE_BERSERK = "Elite Berserk";
 const CHIEFTAINS = "Chieftains";
 const BERSERKERGANG = "Berserkergang";
 
+const horseDisabled = [STABLE, 
+    SCOUT_CAVALRY, LIGHT_CAVALRY, HUSSAR,
+    BLOODLINES, KNIGHT, CAVALIER, PALADIN,
+    CAMEL_RIDER, HEAVY_CAMEL_RIDER, HUSBANDRY,
+    CAVALRY_ARCHER, HEAVY_CAV_ARCHER, SCALE_BARDING_ARMOR,
+    CHAIN_BARDING_ARMOR, PLATE_BARDING_ARMOR, PARTHIAN_TACTICS];
```

Comment:
This looks like it is related to `disableHorses()`, yet it is not currently. Can we somehow resolve that discrepance?

Reply:
Done!